src/Trixi.jl

```diff
@@ -226,7 +226,8 @@ export entropy, energy_total, energy_kinetic, energy_internal, energy_magnetic,
 export lake_at_rest_error
 export ncomponents, eachcomponent
 
-export TreeMesh, StructuredMesh, UnstructuredMesh2D, P4estMesh, T8codeMesh
+export TreeMesh, StructuredMesh, UnstructuredMesh2D, P4estMesh, T8codeMesh,
+       StructuredMeshView
```

Comment:
This is not the same order as in the function arguments. How about same order for consistency?

Reply:
Done.